PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,41 @@
+<!-- You can erase any parts of this template not applicable to your Pull Request. -->
+
+## What's New?
+
+Please include a summary of the changes and the related feature/issue. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+
+## Screenshots (if appropriate):
+
+Please attach any screenshots that would help illustrate the changes.
+
+
+## Type of change
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Documentation update
+
+## Checklist:
+
+- [ ] My code follows the style guidelines of this project
+- [ ] I have performed a self-review of my own code
+- [ ] I have manually tested my code thoroughly
+- [ ] I have added/updated inline documentation for public facing interfaces if relevant
+- [ ] My changes generate no new warnings
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] New and existing integration and unit tests pass locally with my changes
+- [ ] Any dependent changes have been merged and published in downstream modules
+
+
+## Jira Ticket (Internal Only)
```

Comment:
I kept this in because we use it internally for our Jira automations but curious as to whether we want to since it does expose the URL of our Jira instance?

Reply:
I mean it's pretty easy to guess what the instance might be if someone really wanted to look for it, but do you think it's a risk to state it plainly?

Reply:
The alternative of leaving it out is not ideal since the vast majority of PRs are going to be internal

Reply:
Not sure if this is fine or not, hence the question. Its already been exposed for months in the passage-ios and passage-swift repos

Reply:
Yeah, I think given where our support process is at today, I'd rather _not_ include JIira attributes here. These won't always translate to a Jira issue perfectly anyways (it may be added as a product board insight, for example).